.buildkite/scout_ci_config.yml

```diff
@@ -5,5 +5,6 @@ ui_tests:
     - discover_enhanced
     - maps
     - observability_onboarding
+    - painless_lab
```

Comment:
Sadly this is not enough to trigger tests, could you temporarily modify (e.g. add a string) in kbn-scout? It will trigger   all Scout tests including painless_lab